python/python/tests/test_async_client.py

```diff
@@ -7261,6 +7261,27 @@ async def test_copy_database(self, redis_client: GlideClient):
         finally:
             assert await redis_client.select(0) == OK
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_wait(self, redis_client: TGlideClient):
+        key = f"{{key}}-1{get_random_string(5)}"
+        value = get_random_string(5)
+        value2 = get_random_string(5)
+
+        assert await redis_client.set(key, value) == OK
+        if isinstance(redis_client, GlideClusterClient):
+            assert await redis_client.wait(1, 1000) >= 1
+        else:
+            assert await redis_client.wait(1, 1000) >= 0
```

Comment:
Would it be better to use == instead of >= since that's what we have in Java?
```suggestion
            assert await redis_client.wait(1, 1000) == 1
        else:
            assert await redis_client.wait(1, 1000) == 0
```